src/sinks/loki/tests.rs

```diff
@@ -15,8 +15,8 @@ fn generate_config() {
     crate::test_util::test_generate_config::<LokiConfig>();
 }
 
-#[test]
-fn interpolate_labels() {
+#[tokio::test]
+async fn interpolate_labels() {
```

Comment:
It seems like we're running these under tokio now but we didn't have to change anything else... por qué?

Reply:
Ah yeah, that tracks.